lib/srv/desktop/tdp/proto.go

```diff
@@ -675,8 +675,16 @@ func (s SharedDirectoryAnnounce) Encode() ([]byte, error) {
 }
 
 func decodeSharedDirectoryAnnounce(in io.Reader) (SharedDirectoryAnnounce, error) {
+	// TODO(isaiah): The completionID here is a copy-paste error, but we need to keep it
+	// for now in order that the proxy stay compatible with previous versions of the wds.
+	var completionID uint32
```

Comment:
```suggestion
	var discard uint32
```

Let's make it clear that this is not actually a completion ID. (here and below)